platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/shared/CheckoutRequest.java

```diff
@@ -42,12 +44,18 @@
     
     void setCreateNew(boolean isCreateNew);
     
-    BranchCheckoutRequest withCreateNew(boolean isCreateNew);
+    CheckoutRequest withCreateNew(boolean isCreateNew);
 
     /** @return name of branch that will be tracked */
     String getTrackBranch();
 
     void setTrackBranch(String trackBranch);
 
-    BranchCheckoutRequest withTrackBranch(String trackBranch);
+    CheckoutRequest withTrackBranch(String trackBranch);
+    
+    /** @return list of files to checkout */
+    List<String> getFiles();
+    
+    void setFiles(List<String> files);
+
```

Comment:
Not sure what you mean here.... It's a refactor of the class name from BranchCheckoutRequest  to CheckoutRequest